superset/app.py

```diff
@@ -261,8 +261,8 @@ def init_views(self) -> None:
         if self.config["ENABLE_ROW_LEVEL_SECURITY"]:
             appbuilder.add_view(
                 RowLevelSecurityFiltersModelView,
-                "Row Level Security Filters",
-                label=__("Row level security filters"),
+                "Row Level Security",
+                label=__("Row level security"),
```

Comment:
Since all elements related to RLS are under this view, "Filters" can be removed to make the menu narrower.